scripts/migrate_v237_limits.py

```
@@ -0,0 +1,77 @@
+
+print("V2.11 limits migration (dry-run=" + str(dry_run) + ") in processing...")
+failCount = 0
+successCount = 0
+for network in tqdm(networks):
+    variants = get_variants(network['uuid'])
+    for variant in variants:
+        if not dry_run:
+            try:
+                migrate_v237_limits(network['uuid'], variant['num'])
+                successCount += 1
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)